integrated_channels/api/v1/logs/serializers.py

```diff
@@ -66,6 +66,33 @@ class Meta:
         model = LearnerDataTransmissionAudit
         fields = '__all__'
 
+    sync_status = serializers.SerializerMethodField()
+    sync_last_attempted_at = serializers.SerializerMethodField()
+
+    def get_sync_status(self, obj):
```

Comment:
nit: Unit tests for all the branches here?

Reply:
🤔 good idea!